Formula/node.rb

```diff
@@ -38,6 +38,7 @@ class Node < Formula
 
   on_linux do
     depends_on "gcc"
+    depends_on "python@3.9"
```

Comment:
You should make this `uses_from_macos "python3", since: :catalina` instead of `depends_on`, and move it outside of the `on_linux` block.